CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 * Added dump function to XSIAM pack objects to dulicate the files.
 * Fixed an issue where the `contribution_converter` did not support changes made to ApiModules.
 * Added playbook modification capabilities in **TestSuite**.
+* Hotfix to build
```

Comment:
Please move this to the _unreleased_ section, and explain the error fixed (e.g. `Fixed an issue in <process/command> where <what lead to the error>`)